# HG changeset patch
# User Rik <rik@octave.org>
# Date 1522108180 25200
#      Mon Mar 26 16:49:40 2018 -0700
# Branch stable
# Node ID a281de7620d70d51382692e350eaa1652e17b664
# Parent  910804a9d62d8cc58ec8f7f65c978b42a02bd127
doc: Remove deprecated function chop from manual.

* arith.txi: Remove DOCSTRING(chop) from manual.

diff --git a/doc/interpreter/arith.txi b/doc/interpreter/arith.txi
--- a/doc/interpreter/arith.txi
+++ b/doc/interpreter/arith.txi
@@ -249,18 +249,16 @@ cos (pi/2)
 @DOCSTRING(factorial)
 
 @DOCSTRING(factor)
 
 @DOCSTRING(gcd)
 
 @DOCSTRING(lcm)
 
-@DOCSTRING(chop)
-
 @DOCSTRING(rem)
 
 @DOCSTRING(mod)
 
 @DOCSTRING(primes)
 
 @DOCSTRING(list_primes)
 
